src/index.js

```diff
@@ -33,10 +38,12 @@ const PublicDecoratorConfig = DecoratorConfig.extend({
 
 const reduceQueryPropsReturn = queries => t.interface({
   accumulator: t.Any,
-  props: t.interface(
-    queries.reduce((ac, k) => ({ ...ac, [k]: t.Any }), {}),
-    { strict: true }
-  )
+  props: t.interface({
+    ...queries.reduce((ac, k) => ({ ...ac, [k]: t.Any }), {}),
+    readyState: t.interface(queries.reduce((ac, k) => ({ ...ac, [k]: t.interface({
+      waiting: t.Boolean, fetching: t.Boolean, loading: t.Boolean, error: t.maybe(t.Any), ready: t.maybe(t.Boolean)
```

Comment:
you've missed a `{ strict: true }` here.
To simplify this code I would declare outside this block a
```js
const ReadyState = t.interface({
  waiting: t.Boolean,
  fetching: t.Boolean,
  loading: t.Boolean,
  error: t.maybe(t.Any),
  ready: t.maybe(t.Boolean) // why optional??
}, { strict: true });
```